homeassistant/components/device_tracker/aruba.py

```diff
@@ -109,20 +120,57 @@ def get_aruba_data(self):
         elif query == 6:
             _LOGGER.error("Connection timed out")
             return
-        ssh.sendline(self.password)
-        ssh.expect('#')
-        ssh.sendline('show clients')
-        ssh.expect('#')
-        devices_result = ssh.before.split(b'\r\n')
-        ssh.sendline('exit')
 
         devices = {}
-        for device in devices_result:
-            match = _DEVICES_REGEX.search(device.decode('utf-8'))
-            if match:
-                devices[match.group('ip')] = {
-                    'ip': match.group('ip'),
-                    'mac': match.group('mac').upper(),
-                    'name': match.group('name')
-                }
+
+        if self.type == 'AP':
+            ssh.sendline(self.password)
```

Comment:
We shouldn't do any (big) changes to this platform until the device specific interface, I/O and data parsing, is broken out to standalone library published on pypi.

https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#6-communication-with-devices-services

Reply:
Ok, if that is the case i will rewrite to communicate over MQTT